<?xml version="1.0"?>

<!-- launch/pointcloud_to_scan.launch -->
<launch>
    <node name="scan_pub_node" pkg="vision" type="scan_pub_node" output="screen">
        <param name="upper_height" value="0.25" />
        <param name="lower_height" value="0.2" />
        <remap from="input_cloud" to="/ouster/points2" />
        <remap from="output_scan" to="/nav/scan" />
    </node>
</launch>
